Dockerfile

```diff
@@ -1,6 +1,34 @@
 ARG LATEX_IMAGE_TAG=latest
 FROM pandoc/latex:$LATEX_IMAGE_TAG
 
+ENV PLANTUML_VERSION 1.2022.7
+
+RUN apk add --no-cache \
+    py3-pip \ 
+    openjdk17-jre \
```

Comment:
When I try to build the docker image, I get an error saying that `openjdk17-jre` is not available. I am not a Java developer, so I might do something wrong. Does building the image work locally on your side (e.g. `docker build -t rstropek/pandoc-latex:test --build-arg LATEX_IMAGE_TAG=2.18.0 .`)?

Reply:
It is building fine at my side, but i use podman instead of docker, but i don't see any reason that that would make a difference.

podman build -t rstropek/pandoc-latex:test -f Dockerfile

I'll check next week if i can figure out the issue, but in the end the apk package manager should find the openjdk17-jre.